### PR TITLE
[doc] typo fix

### PR DESCRIPTION
**Pull request by stas00**
s/e.g./i.e./ as what follows is not an example, but a "that is" statement + fix language

@sgugger

### CODE REVIEW

File: src/transformers/modeling_utils.py [hunk 1]
```diff
@@ -812,7 +812,7 @@ def from_pretrained(cls, pretrained_model_name_or_path, *model_args, **kwargs):
             output_loading_info(:obj:`bool`, `optional`, defaults to :obj:`False`):
                 Whether ot not to also return a dictionary containing missing keys, unexpected keys and error messages.
             local_files_only(:obj:`bool`, `optional`, defaults to :obj:`False`):
-                Whether or not to only look at local files (e.g., not try doanloading the model).
+                Whether or not to only look at local files (i.e. do not try to download the model).
```
Comment (sgugger) [inline src/transformers/modeling_utils.py:815]:
```suggestion
                Whether or not to only look at local files (i.e., do not try to download the model).
```
Super nit, but O'Reilly made me had those commas after i.e. and e.g. everywhere, so now I like them :-)

**Review by sgugger — APPROVED** [review]
Thanks!

**Comment by stas00** [comment]
Apparently, there is no agreement on the comma: https://www.dailywritingtips.com/comma-after-i-e-and-e-g/, especially after i.e. https://english.stackexchange.com/questions/16172/should-i-always-use-a-comma-after-e-g-or-i-e. 

Therefore it's probably the best to stick to whatever manual of style this projects prefers. And, perhaps, even starting a brief guide for the `transformers` manual of style, where you also include all those stalystic recommendations, such as not documenting `None` as a default for optional objects, the item in question, etc.

My recommendation is that every time a style change is introduced, to change all docs at once, so that the diverse developers will most likely copy/observe how the text is written and follow suite when they create new docs. It's not hard to do:

```
find . -type d -name ".git" -prune -o -type f -exec perl -pi -e "s|e\.g\. |e.g., |g; s|i\.e\. |i.e., |g;" {} \;
```
this one fixes a missing comma.

**Comment by sgugger** [comment]
Thanks @stas00 !

**Comment by sgugger** [comment]
I had missed your comment. We can do the perl magic thing (though check me with privately before applying it as we are scheduling a few big changes today and tomorrow in preparation for the v4.0 so you don't want to do this in the middle of one ;-) ).

As for the documentation, we can add it to our [documentation guide](https://github.com/huggingface/transformers/tree/master/docs#writing-documentation---specification) (which already contains the not defaults to `None`).

**Comment by stas00** [comment]
> I had missed your comment. We can do the perl magic thing (though check me with privately before applying it as we are scheduling a few big changes today and tomorrow in preparation for the v4.0 so you don't want to do this in the middle of one ;-) ).

You can run it any time you know it's a good time. All credits go to perl ;)
 
> As for the documentation, we can add it to our [documentation guide](https://github.com/huggingface/transformers/tree/master/docs#writing-documentation---specification) (which already contains the not defaults to `None`).

That works. Probably start a "grammar style" section.

**Pull request merged**